drawBot/context/__init__.py

```diff
@@ -55,14 +55,16 @@ def getFileExtensions():
     return extensions
 
 
-def getContextOptionsDocs(formatter="* `%s`: %s"):
+def getContextOptionsDocs(formatter="* `%s`: %s") -> list[str]:
     docs = []
     for context in allContexts:
         if context.saveImageOptions:
             ext = ", ".join(context.fileExtensions)
             docs.append("*%s options:*" % ext)
             docs.append("")
-            for key, doc in context.saveImageOptions:
+            for option in context.saveImageOptions:
```

Comment:
I wonder if this change was triggered by the type annotation for saveImageOptions, which I think is wrong. I'll comment there.

Reply:
fixed with 49c19855df09dc1c315f74bc78e5c69496a60f66